bin/dbsv-update.php

```diff
@@ -178,10 +178,10 @@ function update_triggers()
 function dbv_100()  // expands log date to datetime, to enable time logging
 {
     if (sql_field_type('cache_logs', 'date') != 'DATETIME') {
-        sql("ALTER TABLE `cache_logs` CHANGE COLUMN `date` `date` DATETIME NOT NULL");
+        sql('ALTER TABLE `cache_logs` CHANGE COLUMN `date` `date` DATETIME NOT NULL');
     }
     if (sql_field_type('cache_logs_archived', 'date') != 'DATETIME') {
-        sql("ALTER TABLE `cache_logs_archived` CHANGE COLUMN `date` `date` DATETIME NOT NULL");
+        sql('ALTER TABLE `cache_logs_archived` CHANGE COLUMN `date` `date` DATETIME NOT NULL');
     }
```

Comment:
Hrmpf! Siehe der Style-Guide-Vorschlag im Wiki und die interne Diskussion dazu, in der es bislang keine Widersprüche gab zu dem Vorschlag, die SQLs konsistent in " " zu setzen.

Das Blöde an den ' ' ist, dass man dran rumbasteln muss falls man Stringkonstanten in den SQL einfügt (oder dass die Stringkonstanten inkonsistent werden weil man dann plötzlich " " dafür verwendet). Diese SQL hier wird zwar keiner mehr ändern, aber wenn man sie per Copy&Paste als Vorlage verwendet hat man wieder das gleiche Problem (beim Hinzufügen eines Tabellenkommentars).

Mir ist klar dass man so am einfachsten die übrigen Strings auf ' ' bekommt. Aber dann möchte ich nacher zumindest nochmal drübergehen und die SQLs zurückbauen.